mysite/app/views.py

```diff
@@ -397,3 +397,24 @@ def send_tilbud_bookingansvarlig(request, tilbud_id):
     else:
         return redirect('dashboard')
 
+
+'''
+###############
```

Comment:
Ser at vi kan gå for å kommentere koden, men kanskje vi kan komme i felles bestemmelse på møtet om vi har tid og ressurser til å gjøre dette. La det stå inntil videre

Reply:
Litt vel fancy. Kan kommentere det nye eg skriver tenker eg. Lettere å holde orden for egen del, samt finne igjen. Så mange like navn. Kan se kjapt på nokken kommentarer. Evt lage en egen branch for det

Reply:
tenker vi kan kommentere hva ting gjør smått ja for at alle skal skjønne hva som skjer. Dumt at vi ikke startet med det tidligere. men du vet. retrospektiv.